bookshelf/gce/startup-script.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+# Copyright 2015 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START all]
+set -e
+
+# Talk to the metadata server to get the project id
+PROJECTID=$(curl -s "http://metadata.google.internal/computeMetadata/v1/project/project-id" -H "Metadata-Flavor: Google")
+
+# Get the source code
+export HOME=/root
+git config --global credential.helper gcloud.sh
```

Comment:
I mean, you should do:

```
apt-get update
apt-get install -y php5 nginx build-essential php5-fpm git
```

This way it won't break if the default base image changes right?